content/en/real_user_monitoring/session_replay/mobile/setup_and_configuration.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Mobile Session Replay Setup and Configuration
+kind: documentation
+description: Describes how to setup and configure Mobile Session Replay
+aliases:
+further_reading:
+    - link: '/real_user_monitoring/session_replay/mobile'
+      tag: Documentation
+      text: Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay/mobile/app_performance'
+      tag: Documentation
+      text: How Mobile Session Replay Impacts App Performance
+    - link: '/real_user_monitoring/session_replay/mobile/privacy_options'
+      tag: Documentation
+      text: Mobile Session Replay Privacy Options
+    - link: '/real_user_monitoring/session_replay/mobile/troubleshooting'
+      tag: Documentation
+      text: Troubleshoot Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay'
+      tag: Documentation
+      text: Session Replay
+---
+
+## Setup
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+All Session Replay SDK versions can be found in the [Maven Snapshots Repository][1].
+
+To set up Mobile Session Replay for Android:
+
+1. Make sure you've [setup and initialized the Datadog Android RUM SDK][2].
```

Comment:
We need to mention here that they need to enable the view/action RUM tracking strategies for Session Replay to work.

Reply:
View tracking is minimal requirement. Actions - not necessarily.

Reply:
@mariusc83 Can you specify where and how to enable this setting? Feel free to make a code suggestion!

Reply:
Or is it enough to say:

```suggestion
1. Make sure you've [setup and initialized the Datadog Android RUM SDK][2] with views instrumentation enabled.
```